src/libs/UnreadIndicatorUpdater/index.js

```diff
@@ -7,13 +7,14 @@ import * as ReportUtils from '../ReportUtils';
 const reports = {};
 
 /**
- * Updates the title and favicon of the current browser tab
- * and Mac OS or iOS dock icon with an unread indicator.
+ * Updates the title and favicon of the current browser tab and Mac OS or iOS dock icon with an unread indicator.
+ * Note: We are throttling this since listening for report changes can trigger many updates depending on how many reports
+ * a user has and how often they are updated.
  */
 const throttledUpdatePageTitleAndUnreadCount = _.throttle(() => {
     const totalCount = _.filter(reports, ReportUtils.isUnread).length;
     updateUnread(totalCount);
-}, 1000, {leading: false});
+}, 100, {leading: false});
```

Comment:
Updated this since 1 second is a pretty long time and we are mainly throttling to avoid quick blasts of reports triggering the update when you maybe have hundreds of them that are getting read from Onyx on app init or something.